src/templates/mt5/economic_calendar.html.tt

```diff
@@ -0,0 +1,21 @@
+<div class="primary-color-dark cfd-header">
+    <div class="fx-panel center-text container">
+        <h1 class="content-inverse-color">[% l('Economic Calendar') %]</h1>
+        <p class="content-inverse-color">[% l('Check out the latest economic events and indicators from all over the world. All data are collected and updated in real time.') %]</p>
+    </div>
+</div>
+
+<div class="container">
+    <p>[% l('You can monitor macroeconomic indicators straight from the MetaTrader 5 platform using release time labels displayed on <a href="[_1]" target="_blank">financial symbol charts</a>.', request.url_for('/download/metatrader5_help.pdf')) %]</p>
```

Comment:
File name used here `metatrader5_help.pdf` doesn't match the actual file name `MetaTrader5_Help.pdf`